docs/app/Component/ComponentDescription.js

```diff
@@ -59,9 +60,7 @@ class ComponentDescription extends Component {
           </code>
         </div>
         {this.isSemanticComponent && semanticDocsLink}
-      </div>
+      </Segment>
```

Comment:
The `ComponentDescription` is the top section of every component doc section.  It is now a basic segment so that it has a bottom whitespace.  This is needed to keep whitespace between the header and prop types table segment.